[2025-02-28T11:06:05.146+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T11:03:00+00:00 [queued]>
[2025-02-28T11:06:05.160+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T11:03:00+00:00 [queued]>
[2025-02-28T11:06:05.161+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T11:06:05.182+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 11:03:00+00:00
[2025-02-28T11:06:05.188+0000] {standard_task_runner.py:60} INFO - Started process 9548 to run task
[2025-02-28T11:06:05.192+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-02-28T11:03:00+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpgwhtisbu']
[2025-02-28T11:06:05.196+0000] {standard_task_runner.py:88} INFO - Job 210: Subtask load_to_postgres_file
[2025-02-28T11:06:05.270+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T11:03:00+00:00 [running]> on host 335ace54b661
[2025-02-28T11:06:05.387+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T11:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T11:03:00+00:00'
[2025-02-28T11:06:05.391+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T11:06:05.393+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T11:06:05.413+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T11:06:06.721+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/18-06-03.csv' từ MinIO
[2025-02-28T11:06:06.722+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T11:06:06.724+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-02-28T11:06:06.816+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T11:06:06.851+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T110300, start_date=20250228T110605, end_date=20250228T110606
[2025-02-28T11:06:06.897+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T11:06:06.919+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
